src/app/products/charges/create-charge/create-charge.component.html

```diff
@@ -122,27 +120,33 @@
             </mat-form-field>
 
             <mat-form-field fxFlex="48%" *ngIf="chargeForm.controls.chargeTimeType.value === 7 || chargeForm.controls.chargeTimeType.value === 11">
-              <mat-label>Repeat Every {{ repeatEveryLabel }}</mat-label>
+              <mat-label>Repeat Every ({{ repeatEveryLabel }})</mat-label>
               <input type="number" step="1" matInput required formControlName="feeInterval">
-              <mat-error *ngIf="chargeForm.controls.feeInterval.hasError('required')">
-                Repeat Every is <strong>required</strong>
+              <mat-error *ngIf="chargeForm.controls.feeInterval.hasError('required') && !chargeForm.controls.feeInterval.hasError('pattern')">
+                Repeat Every ({{ repeatEveryLabel }}) is <strong>required</strong>
               </mat-error>
-              <mat-error *ngIf="chargeForm.controls.feeInterval.hasError('min') || chargeForm.controls.feeInterval.hasError('max')">
-                Repeat Every <strong>must be between 1 and 12 months</strong>
+              <mat-error *ngIf="(chargeForm.controls.feeInterval.hasError('min') || chargeForm.controls.feeInterval.hasError('max')) && !chargeForm.controls.feeInterval.hasError('pattern')">
```

Comment:
I decided to only show the range error if there isn't a pattern error to prevent double errors appearing.

Reply:
What is the possibility of them occurring together?

Reply:
They occur together when the inputted number is negative or 0. There is no point in displaying both errors, because if a number is negative or 0, they are also not between 1 and 12, and the two errors will be saying the same thing. The error should only appear if the number is already positive and an integer, but not between 1 and 12.